src/pretix/presale/views/customer.py

```diff
@@ -96,11 +108,23 @@ def get_form_kwargs(self):
 
     def get_success_url(self):
         url = self.get_redirect_url()
-        if getattr(self.request, 'event_domain', False):
-            default_url = '/'
-        else:
-            default_url = eventreverse(self.request.organizer, 'presale:organizer.customer.profile', kwargs={})
-        return url or default_url
+
+        if not url:
+            return eventreverse(self.request.organizer, 'presale:organizer.customer.profile', kwargs={})
+
+        if self.request.GET.get("request_cross_domain_customer_auth") == "true":
```

Comment:
As discussed we should ignore `request_cross_domain_customer_auth` when it is not on a custom organizer/event domain.